oneview_redfish_toolkit/api/scmb.py

```diff
@@ -101,7 +106,35 @@ def get_cert():
 
     with open(_oneview_ca_path(), 'w+') as f:
         f.write(cert)
-    # Generate scmb Cert:
+
+    global certs
+
+    try:
+        # Checks if certificate exists
+        certs = ov_client.certificate_rabbitmq.get_key_pair(
+            'default')
+
+    except HPOneViewException as e:
+        if e.oneview_response["errorCode"] in NOT_FOUND_ONEVIEW_ERRORS:
+            logging.info('Generating new SCMB certs in Oneview...')
+            _generate_scmb_certificate(ov_client)
+            certs = ov_client.certificate_rabbitmq.get_key_pair(
+                'default')
+        else:
+            logging.exception("Unexpected error")
+            raise
+
+    logging.info('SCMB certs already generated in Oneview. '
+                 'Getting certs...')
```

Comment:
This log message should be in the end of the try block

Reply:
I would also change to "RabbitMQ certificate and key pair retrieved from OneView."